src/main/java/menion/android/whereyougo/gui/activity/CartridgeDetailsActivity.java

```diff
@@ -96,8 +97,9 @@ public void onCreate(Bundle savedInstanceState) {
             loc1.setLongitude(MainActivity.cartridgeFile.longitude);
             Guide guide = new Guide(MainActivity.cartridgeFile.name, loc1);
             A.getGuidingContent().guideStart(guide);
-            MainActivity.callGudingScreen(CartridgeDetailsActivity.this);
-            CartridgeDetailsActivity.this.finish();
+            Intent intent = new Intent(CartridgeDetailsActivity.this, GuidingActivity.class);
+            startActivity(intent);
+            finish();
```

Comment:
Please shorten in the same way in line 90.

Reply:
Can we please write all of the style rules down or enforce them. I find line 102 logical, even more in the light of what I have done in line 101. I personally find line 91 somehow illogical as I am literally just calling a method of an object. Prefixing that is just a wast of characters imho. But again. You are the boss here.

Reply:
I really meant `shorten` - in L91.
To `finish();` as you had done in L102.

Reply:
Ahhh okay that makes a lot of sense. Sorry for the misunderstanding. Correction is done!